README.md

```diff
@@ -101,11 +90,11 @@ Run it using the `dockstore` CLI:
 ```
 Usage:
 # fetch WDL
-$> dockstore tool wdl --entry quay.io/collaboratory/dockstore-tool-bamstats > Dockstore.wdl
+$> dockstore workflow wdl --entry github.com/dockstore/dockstore-tool-bamstats/wdl:feature/update > bamstats.wdl
 # make a runtime JSON template and edit it (or use the content of test.wdl.json above)
-$> dockstore tool convert wdl2json --wdl Dockstore.wdl > Dockstore.json
+$> dockstore tool convert wdl2json --wdl bamstats.wdl > Dockstore.json
```

Comment:
This line should be `dockstore workflow convert` for consistency's sake.

Funnily enough when testing this I found some behavior that perhaps we don't want...
```
>dockstore workflow convert wdl2json --wdl variantcalling.wdl > eireori.json
>cat eireori.json
Current version : 1.12.0
You have the most recent stable release.
If you wish to upgrade to the latest unstable version, please use the following command:
   dockstore --upgrade-unstable
{
  "DraftTuberculosisAssembly.clockwork.cpu": "Int (optional, default = 4)",
  "DraftTuberculosisAssembly.skip_trimming": "Boolean (optional, default = false)",
  "DraftTuberculosisAssembly.genome": "File",
  "DraftTuberculosisAssembly.clockwork.addldisk": "Int (optional, default = 1)",
  "DraftTuberculosisAssembly.clockwork.memory": "Int (optional, default = 8)",
  "DraftTuberculosisAssembly.genome_index": "Array[File]? (optional)",
  "DraftTuberculosisAssembly.clockwork.preempt": "Int (optional, default = 2)",
  "DraftTuberculosisAssembly.illumina_reads": "Array[File]? (optional)",
  "DraftTuberculosisAssembly.clockwork.retries": "Int (optional, default = 3)"
}
```

Reply:
I have a vague recollection of the Dockstore CLI inappropriately putting such text in an output like this before, and a ticket was open to suppress version information when outputting certain files, but I can't find it now.